ssl/ssl_test.cc

```diff
@@ -6598,9 +6595,52 @@ TEST_P(MultipleCertificateSlotTest, AutomaticSelection) {
       {certificate_key_param().corresponding_sigalg}, SSL_PKEY_ED25519, true);
 }
 
+TEST_P(MultipleCertificateSlotTest, AutomaticSelectionCipherAuth) {
+  if ((version < TLS1_2_VERSION && slot_index == SSL_PKEY_ED25519) ||
+      version >= TLS1_3_VERSION) {
+    // ED25519 is not supported in versions prior to TLS1.2.
+    // TLS 1.3 not have cipher-based authentication configuration.
+    return;
```

Comment:
minor nit here, L6672, and L6645: `GTEST_SKIP()` instead of returning.

Reply:
Ah yes forgot about that, I went ahead and did it for all tests we're already slightly optimizing.